packages/types/src/index.ts

```diff
@@ -225,9 +225,17 @@ export interface SentryResponse {
 
 /** JSDoc */
 export interface TransportOptions {
+  [key: string]: any;
```

Comment:
Do we really need this?

Reply:
Otherwise we cannot assign `httpProxy` and friends to `transportOptions` and I wanted it to be consistent with other SDKs and not ask users to pass it directly there, eg.:

```js
init({
  transportOptions: {
    httpProxy: 'http://something:8008'
  }
})
```

(which would be a better option imo, but it's not how it works in other SDKs)